examples/storage/couchdb/images/readiness_probe/join_cluster.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+"""
+    Usage:  join_cluster.py dbname dbname ...
+"""
+from dns import resolver, exception
+import requests
+import logging
+import socket
+import sys
+import os
+
+
+def main(system_dbs):
+    pod_hostnames = discover_pod_fq_hostnames()
+    logging.warning('Using the discovered host names: {0}'.format(pod_hostnames))
+
+    cluster_admin = (os.environ['CLUSTER_ADM_USER'].rstrip(), os.environ['CLUSTER_ADM_PASS'].rstrip())
+    populate_nodes_db(pod_hostnames, cluster_admin)
+    create_system_dbs(cluster_admin, system_dbs)
```

Comment:
You really only want to execute `create_system_dbs` after the cluster is fully-connected, right? I'm worried about the possibility of creating conflicted system DBs here by executing this command for each Pod. Is there any way to defer it until all members of the initial Pet Set are up and running?